dark-mode/README.md

```diff
@@ -0,0 +1,66 @@
+## Minimalistic rEFInd theme
```

Comment:
Instead of duplicating the license file, can we update the main readme to indicate how to use the darkmode version? :)